Wage/Extensions/NotificationExtensions.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  NotificationExtensions.swift
+//  Wage
+//
+//  Created by Patrick Rugebregt on 20/02/2022.
+//
+
+import Foundation
+
+extension Notification.Name {
+    static let shareUser: Notification.Name = Notification.Name("shareUser")
+}
```

Comment:
We don't use notifications anymore. Please update
```suggestion
} Remove notifications
```